index.html

```diff
@@ -115,6 +115,167 @@ <h1>Fresno Bee</h1>
+							  <p>lorem28</p>
+						  </div>
+						  <div class="col-sm-3 col-xs-6 food">
+							  <img src="img/meat.jpg"class=" img-thumbnail img-responsive">
+							  <h3>Appetizer 3</h3>
+							  <p>lorem28</p>
+						  </div>
+						  <div class="col-sm-3 col-xs-6 food">
+							  <img src="img/bruschetta.jpg" class=" img-thumbnail img-responsive">
+							  <h3>Appetizer 4</h3>
+							  <p>lorem28</p>
+						  </div>
+				  </div>
+					
+					  <div id="lunch" class="tab-pane fade">
+						  <div class="col-sm-3 col-xs-6 food">
+							  <img src="img/lunch1.jpg" alt="" class="img-thumbnail img-responsive">
+							  <h3>Lunch 1</h3>
+							  <p>lorem28</p>
+						  </div>
+						  <div class="col-sm-3 col-xs-6 food">
+							  <img src="img/lunch2.jpg" alt="" class="img-thumbnail img-responsive">
+							  <h3>Lunch 2</h3>
+							  <p>lorem28</p>
+						  </div>
+						  <div class="col-sm-3 col-xs-6 food">
+							  <img src="img/lunch3.jpg" alt="" class="img-thumbnail img-responsive">
+							  <h3>Lunch 3</h3>
+							  <p>lorem28</p>
+						  </div>
+						  <div class="col-sm-3 col-xs-6 food">
+							  <img src="img/lunch4.jpg" alt="" class="img-thumbnail img-responsive">
+							  <h3>Lunch 4</h3>
+							  <p>lorem28</p>
+						  </div>
+					</div> 
+					  
+					  <div id="dinner" class="tab-pane fade">
+						  <div class="col-sm-3 c0l-xs-6 food">
+							  <img src="img/dinner1.jpg" alt="" class="img-responsive img-thumbnail">
+							  <h3>Dinner 1</h3>
+							  <p>lorem28</p>
+						  </div>
+						  <div class="col-sm-3 c0l-xs-6 food">
+							  <img src="img/dinner2.jpg" alt="" class="img-responsive img-thumbnail">
+							  <h3>Dinner 2</h3>
+							  <p>lorem28</p>
+						  </div>
+						  <div class="col-sm-3 c0l-xs-6 food">
+							  <img src="img/dinner3.jpg" alt="" class="img-responsive img-thumbnail">
+							  <h3>Dinner 3</h3>
+							  <p>lorem28</p>
+						  </div>
+						  <div class="col-sm-3 c0l-xs-6 food">
+							  <img src="img/dinner4.jpg" alt="" class="img-responsive img-thumbnail">
+							  <h3>Dinner 4</h3>
+							  <p>lorem28</p>
+						  </div>
+					  </div>
+					  <div id="dessert" class="tab-pane fade">
+						  <div class="col-sm-3 col-xs-6 food">
+							  <img src="img/dessert1.jpg" alt="" class="img-thumbnail img-responsive">
+							  <h3>Dessert 1</h3>
+							  <p>lorem28</p>
+						  </div>
+						  <div class="col-sm-3 col-xs-6 food">
+							  <img src="img/dessert2.jpg" alt="" class="img-thumbnail img-responsive">
+							  <h3>Dessert 2</h3>
+							  <p>lorem28</p>
+						  </div>
+						  <div class="col-sm-3 col-xs-6 food">
+							  <img src="img/dessert3.jpg" alt="" class="img-thumbnail img-responsive">
+							  <h3>Dessert 3</h3>
+							  <p>lorem28</p>
+						  </div>
+						  <div class="col-sm-3 col-xs-6 food">
+							  <img src="img/dessert4.jpg" alt="" class="img-thumbnail img-responsive">
+							  <h3>Dessert 4</h3>
+							  <p>lorem28</p>
```

Comment:
All your lorems need to be tabbed out. Due to this, I am requesting changes, thanks!